goss/goss-kernel-params.yaml

```diff
@@ -6,22 +6,19 @@ kernel-param:
     value: "1"
   net.bridge.bridge-nf-call-ip6tables:
     value: "1"
-{{if eq .Vars.OS "photon-3"}}
+{{if contains "1.24" .Vars.kubernetes_version}}
```

Comment:
I think this might fail in `1.24.9` Ubuntu as we expect these parameters value to be set to `1` in `1.24.9` Ubuntu. 🤔

Reply:
resolved with latest commit